packages/legacy/core/App/container-api.ts

```diff
@@ -78,6 +80,7 @@ export interface TokenMapping {
   [TOKENS.COMP_BUTTON]: Button
   [TOKENS.OBJECT_ONBOARDINGCONFIG]: ScreenOptionsType
   [TOKENS.UTIL_LOGGER]: BaseLogger
+  [TOKENS.UTIL_OCA_RESOLVER]: DefaultOCABundleResolver
```

Comment:
Shouldn't this be the interface for the resolver (OCABundleResolverType), like it is on current ConfigurationContext?

Reply:
Fixed